src/internal/pfsdb/commits.go

```diff
@@ -826,15 +916,19 @@ func (iter *CommitIterator) getCommitInfosForPageUntilCapacity(ctx context.Conte
 	return nil
 }
 
-func listCommitPage(ctx context.Context, tx *pachsql.Tx, limit, offset int, filter CommitListFilter, rev bool) ([]Commit, error) {
+func listCommitPage(ctx context.Context, tx *pachsql.Tx, limit, offset int, filter CommitListFilter, reverse, revision bool) ([]Commit, error) {
```

Comment:
Maybe add a comment here or where CommitListFilter is defined, warning of possible SQL injection for user controller filter keys.

Reply:
And values now that I'm looking.